public/app/features/canvas/runtime/frame.tsx

```diff
@@ -151,6 +163,11 @@ export class FrameState extends ElementState {
         this.scene.save();
         this.reinitializeMoveable();
         break;
+      case LayerActionID.MoveTop:
+      case LayerActionID.MoveBottom:
+        element.parent?.doMove(element, action);
```

Comment:
Small thing but might be a little cleaner if we split this up into two helper functions (`moveToTop` and `moveToBottom`) and handle these cases separately